### PR TITLE
Added U2F animation

### PR DESCRIPTION
**Pull request by mitsuhiko**
**Tapped state:**

tapped

### CODE REVIEW

File: src/sentry/static/sentry/app/components/u2finterface.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const U2fInterface = React.createClass({
       isSupported: null,
       formElement: null,
       challengeElement: null,
+      hasBeenTapped: false,
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/u2finterface.jsx]: `wasTapped`?

File: src/sentry/static/sentry/app/components/u2finterface.jsx [hunk 2]
```diff
@@ -49,10 +50,14 @@ const U2fInterface = React.createClass({
...
     }
     promise.then((data) => {
-      this.state.responseElement.value = JSON.stringify(data);
-      if (!this.props.onTap || this.props.onTap()) {
-        this.state.formElement.submit();
-      }
+      this.setState({
+        hasBeenTapped: true
+      }, () => {
+        this.state.responseElement.value = JSON.stringify(data);
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/u2finterface.jsx]: Why is this not modified in the preceding `setState` call?

**Comment by ckj** [comment]
Added a gif of the 'tapped' state

**Comment by dcramer** [comment]
Anything pending here?

**Comment by ckj** [comment]
🚢

**Pull request merged**